tox.ini

```diff
@@ -57,4 +57,5 @@ commands =
 # Excluding our upstream submodule, and our vendored f5 configuration script.
 
 ignore=F403,H303,R
-exclude=openstack-ansible/*,f5-config.py
```

Comment:
no, but maybe should be a different commit.  This and the commit to f5-config get us fully compliant.

F403 prevents `from foo import *` and we don't do that anywhere I can see.
H303 Do not use wildcard * import (*), don't think we do that either...